src/xcode/ENA/ENA/Source/Scenes/ContactDiary/StoreV2/ContactDiaryStoreV2.swift

```diff
@@ -0,0 +1,995 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+import FMDB
+import Combine
+
+// swiftlint:disable:next type_body_length
+class ContactDiaryStoreV2: DiaryStoring, DiaryProviding {
+
+	static let encriptionKeyKey = "ContactDiaryStoreEncryptionKey"
+
+	// MARK: - Init
+
+	init?(
+		databaseQueue: FMDatabaseQueue,
+		schema: ContactDiaryStoreSchemaV2,
+		key: String,
+		dateProvider: DateProviding = DateProvider(),
+		latestDBVersion: Int
+	) {
+		self.databaseQueue = databaseQueue
+		self.key = key
+		self.dateProvider = dateProvider
+		self.schema = schema
+		self.latestVersion = latestDBVersion
+		
+		guard case .success = openAndSetup() else {
+			return nil
+		}
+
+		guard case .success = cleanup() else {
+			return nil
+		}
+
+		var updateDiaryResult: DiaryStoringVoidResult?
+		databaseQueue.inDatabase { database in
+			updateDiaryResult = updateDiaryDays(with: database)
+		}
+		guard case .success = updateDiaryResult else {
+			return nil
+		}
+		registerToDidBecomeActiveNotification()
+	}
+	
+	// MARK: - Protocol DiaryProviding
+
+	var diaryDaysPublisher = CurrentValueSubject<[DiaryDay], Never>([])
+
+	func export() -> Result<String, SQLiteErrorCode> {
+		var result: Result<String, SQLiteErrorCode>?
+
+		databaseQueue.inDatabase { database in
+			Log.info("[ContactDiaryStore] export entries.", log: .localData)
+			var contentHeader = "Kontakte der letzten 14 Tage (%@ - %@)\nDie nachfolgende Liste dient dem zuständigen Gesundheitsamt zur Kontaktnachverfolgung gem. § 25 IfSG."
```

Comment:
Shouldn't we localize this – even if some people think there are only German speaking entities in this country?

```suggestion
			var contentHeader = NSLocalizedString("some.key", tableName: nil, bundle: nil, value: "Kontakte der letzten 14 Tage (%@ - %@)\nDie nachfolgende Liste dient dem zuständigen Gesundheitsamt zur Kontaktnachverfolgung gem. § 25 IfSG.", comment: "Currently not translated!")

```

Reply:
I guess we are not localizing because this whole export feature is only available for Germany "at least for now", same goes for date formatting we are only using German Local